projects/extension/sql/incremental/007-secret-permissions-dump-filter.sql

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
lets add a comment in `002-secret_permissions.sql` that the config dump settings are changed in this file

Reply:
it will generate a warning on upgrade since that file has already shipped and is now frozen. that okay with you?